src/settings/OkapiConsole/Modules.js

```diff
@@ -33,7 +33,88 @@ function maybeRenderInterfaces(detail, tag, isPermissions) {
   );
 }
 
-function ModuleDetail({ detail }) {
+
+function undeploy(okapiKy, callout, id, instId) {
+  return okapiKy.delete(`_/discovery/modules/${id}/${instId}`)
```

Comment:
Just so you are aware -- Okapi will allow you to do this even if the module is in use by a tenant, so you can easily break an environment this way.